### PR TITLE
Adding the `important-links` component

### PR DESCRIPTION
**Pull request by vazquez**
This PR and https://github.com/mozilla/mozmaker-templates/pull/48 go hand-in-hand.

### CODE REVIEW

File: src/scss/custom/_important-links.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  margin: 0 -10px 30px -10px;
+  @media screen and (min-width: 544px) {
+    flex-direction: row;
+  }
+
+  li {
+    list-style-type: none;
+    flex: 1;
+    margin: 10px;
+    height: 100px;
```
Comment (gvn) [inline src/scss/custom/_important-links.scss]: Hardcoding height isn't a good idea for responsive layouts.

Comment (gvn) [inline src/scss/custom/_important-links.scss]: Could do min-height though

**Comment by gvn** [comment]
clear

**Comment by vazquez** [comment]
HAHAHAHAHAHAHA

**Pull request merged**